pull_request_template.md

```diff
@@ -1,80 +1,13 @@
 # Description
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. Explain both the expected functionality and the edge cases. 
+*The intended change, motivation and consequences.*
```

Comment:
Maybe add a few suggestions of references to add (here or on notes) like:
* Jira ticket (if applicable)
* Feature reference (I usually do this as a prefix in the title [Feature A])
* Screenshot (also suggest adding a PD or a PM to chime in)

Reply:
Good suggestions, though I think they should all go in the new linked "PR Guidelines" document to keep the template minimal and general. 

> Jira ticket (if applicable)

I think this is covered by "motivation". As a convention, I've see other teams put it in the title. Something to consider.

> Feature reference

This goes in the title.... not sure if it is clear putting it in the description as a suggestion.

> Screenshot

Hopefully this is obvious enough.